auth/service_provider.go

```diff
@@ -32,8 +32,10 @@ func (database *ServiceProvider) Boot(app foundation.Application) {
 }
 
 func (database *ServiceProvider) registerCommands(app foundation.Application) {
-	app.MakeArtisan().Register([]contractconsole.Command{
-		console.NewJwtSecretCommand(app.MakeConfig()),
-		console.NewPolicyMakeCommand(),
-	})
+	if artisanFacade := app.MakeArtisan(); artisanFacade != nil {
```

Comment:
When `artisanFacade == nil`, we should print a warning, other ServicePrivoders are the same. And I think it's a good time to implement https://github.com/goravel/goravel/issues/495.

```suggestion
	if artisanFacade := app.MakeArtisan(); artisanFacade == nil {
	  color.Red.Println("****")
	  return
	} 
```

Reply:
I didn't give warning because of this issue https://github.com/goravel/goravel/issues/423